docs/source/checks/vision/model_evaluation/plot_class_performance.py

```diff
@@ -94,6 +106,18 @@
 check = ClassPerformance(show_only='best')
 check.run(train_ds, test_ds, yolo)
```

Comment:
Let's use this PR to update all check runs to be saved in "result" (like in line 130)